### PR TITLE
Update sell page mail_to I18n (and UK translations)

### PR DESCRIPTION
**Pull request by lin-d-hop**
Small update to internationalise the mail_to on the sell page to use the instance email rather than a hardcoded aus email.

Translation updates for the sell page for en-GB.yml also in this PR.

@daniellemoorhead would you ask someone to take a look and merge if happy?

### CODE REVIEW

**Comment by daniellemoorhead** [comment]
Hiya @lin-d-hop I've assigned this to @mkllnk ...but it's saying that all checks have failed, so does something need to be done to get it to pass before he looks at it?

**Comment by lin-d-hop** [comment]
@daniellemoorhead Tests are passing :-)
@mkllnk Thanks for taking a look!

File: app/views/home/sell.html.haml [hunk 1]
```diff
@@ -44,7 +44,6 @@
 
         %p
           = t :sell_embed
-          %a{href: "hello@openfoodnetwork.org".reverse, target: '_blank', mailto: true}
-            = t :sell_ask_services
+          = mail_to ContentConfig.footer_email, t(:sell_ask_services)
```
Comment (mkllnk) [inline app/views/home/sell.html.haml]: Is the `reverse` missing here? We use it to obfuscate the email address. The `mailto` attribute triggers some javascript to un-reverse it again for the user.

**Comment by lin-d-hop** [comment]
Hi @mkllnk 
Made the change to the reverse that you suggested. Seems that using mail_to here the .reverse is not re-reversed (versed?). Testing locally we just end up with a backwards email address and delivery failures. Is it ok to leave the .reverse out?

I think the merge conflict is coming from the fact the config/locales/en_GB.yml is deleted in this PR. This was added by me incorrectly after downloading from transfiex and not changing to en-GB like the existing file. I'm not sure how to resolve this without merging the deleted file on master... how is best for me to resolve this?

**Comment by lin-d-hop** [comment]
Replaced by #1176

**Comment by lin-d-hop** [comment]
@mkllnk Merge conflicts now fixed.

**Comment by mkllnk** [comment]
@lin-d-hop Thank you for solving the conflicts. I added the `mailto` attribute to restore the email address when clicked. We could probably put that into a helper like `mask_mail_to`.

**Comment by sstead** [comment]
Testing notes (for my reference)... 
Initial feedback...
image
This was corrected, now emails of both formats work, in all browsers, written forwards :)

**Pull request merged**